homeassistant/components/mqtt/climate.py

```diff
@@ -273,14 +277,21 @@
 
 
 async def async_setup_platform(
-    hass: HomeAssistant, async_add_entities, config: ConfigType, discovery_info=None
```

Comment:
Why are the parameters in a different order here?

Reply:
I assume it was a typo, and I have adjusted it below in `_async_setup_entity`

Reply:
I agree, looks like a typo.

But I didn't see your edit to `_async_setup_entity` in the diff.

Reply:
I didn't need to change it in the `_async_setup_entity` definition.
I had to adjust it in the `_async_setup_entity` call within `async_setup_platform` on line 287 (old line 280)

Reply:
Ok.

From your message I thought you were modifying the function.

I assume the reason why you didn't change the order to match the other functions was because `config_entry` is optional here but in other functions it's compulsory (so has to come first).

Looks good 👍